examples/source/1.components/amp-date-picker.html

```diff
@@ -69,6 +69,14 @@
     margin-left: var(--space-2);
   }
 
+  .DayPicker_weekHeader {
+    top: 60px;
```

Comment:
Please check the stand alone preview and the playground preview.
It's only together with the styles from amp.dev that it was messed up.
The changes fixes the inline preview, but introduces problems in the stand alone example.
When we introduce styles that fix the inline preview, we should add a comment, to make clear that it is only needed for the inline preview on amp.dev
If possible it would be better to change the site styles so that the effects on the inline preview is removed.

Reply:
should be fixed now